### PR TITLE
Add org.freedesktop.Sdk.Extension.node12

### PR DESCRIPTION
**Pull request by mbooth101**
This is almost identical to the node10 extension ( see https://github.com/flathub/org.freedesktop.Sdk.Extension.node10 ) just updated to version 12.x, the newest LTS version of node.

### CODE REVIEW

File: org.freedesktop.Sdk.Extension.node12.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+cleanup:
+  - '/bin/pip*'
+  - '/bin/python*config'
+  - '/bin/smtpd.py'
+  - '/lib/pkgconfig'
+  - '/share/doc'
+  - '/share/man'
+  - '/share/systemtap'
+modules:
+  - shared-modules/python2.7/python-2.7.json
```
Comment (gasinvein) [inline org.freedesktop.Sdk.Extension.node12.yaml]: Does Node.js 12 still require Python 2?

Comment (mbooth101) [reply org.freedesktop.Sdk.Extension.node12.yaml]: For building, yes -- use of python 3 is still experimental (according to the readme)

**Comment by mbooth101** [comment]
bot, build org.freedesktop.Sdk.Extension.node12

**Pull request closed without merging**